frontend/src/router/router.tsx

```diff
@@ -66,6 +67,14 @@ const router = createBrowserRouter([
             </PublicRoute>
         ),
     },
+    {
+        path: "FulFilPurchaseRequest",
```

Comment:
Would you as user wanna navigate to http://ezTix/FulfilPurchaseRequest ? Perhaps rename the route to something you can think of that is simpler